recipe/1.16/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.16" %}
```

Comment:
If you need an old version we should create a branch for it.
Another directory won't work.

Reply:
Thanks, sorry I realized after I created this that `gsl-1.16` is already available, I just didn't see it listed on anaconda.org.